### PR TITLE
Openshift conformance excluded test suite list as non admin

### PR DESCRIPTION
**Pull request by mbpavan**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -126,12 +126,24 @@ The container image includes a few lists proven to be successfully executed with
 * [openshift-conformance](tests-lists/openshift-conformance.txt). Based on the openshift/conformance tests suite (`openshift-tests run openshift/conformance --dry-run | grep -v -i "clusteradmin"` and extracted the ones that passed with non cluster-admin permissions)
 * [kubernetes-conformance](tests-lists/kubernetes-conformance.txt).  Based on the kubernetes/conformance tests suite (`openshift-tests run kubernetes/conformance --dry-run | grep -v -i "clusteradmin"` and extracted the ones that passed with non cluster-admin permissions)
 * [openshift-network-stress](tests-lists/openshift-network-stress.txt).  Based on the openshift/network/stress tests suite (`openshift-tests run  openshift/network/stress --dry-run | grep -v -i "clusteradmin"` and extracted the ones that passed with non cluster-admin permissions)
+* [openshift-conformance-excluded](tests-lists/openshift-conformance-excluded.txt).  Based on the openshift/conformance-excluded tests suite (`openshift-tests run  openshift/conformance-excluded --dry-run | grep -v -i "clusteradmin"` and extracted the ones that passed with non cluster-admin permissions)
```
Comment (e-minguez) [inline README.md]: can you remove the blank space between run and openshift? Thanks.

**Review by e-minguez — CHANGES_REQUESTED** [review]
Can you add those new ones to the all.txt?

File: README.md (hunk 1 above)
Comment (mbpavan) [reply README.md]: Done!

**Comment by mbpavan** [comment]
Done. Added them to all.txt

File: tests-lists/all.txt [hunk 2]
```diff
@@ -139,8 +139,6 @@
 "[sig-cli] Kubectl client Kubectl label should update the label on a resource  [Conformance] [Suite:openshift/conformance/parallel/minimal] [Suite:k8s]"
 "[sig-cli] Kubectl client Kubectl logs should be able to retrieve and filter logs  [Conformance] [Disabled:sig-cli] [Suite:k8s]"
 "[sig-cli] Kubectl client Kubectl patch should add annotations for pods in rc  [Conformance] [Suite:openshift/conformance/parallel/minimal] [Suite:k8s]"
-"[sig-cli] Kubectl client Kubectl replace should update a single-container pod's image  [Conformance] [Disabled:sig-cli] [Suite:k8s]"
```
Comment (e-minguez) [inline tests-lists/all.txt:143]: why are those tests deleted?

**Comment by mbpavan** [comment]
I will send a new PR after rebase.

**Pull request closed without merging**